.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,10 +8,12 @@ to the issue. -->
 
 - [ ] Commit sequence broadly makes sense and commits have useful messages
 - [ ] New tests are added if needed and existing tests are updated
-- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
+- [ ] All visible changes are prepended to the latest section of a `CHANGELOG.md` for the affected packages.
+      **_New section is never added with the code changes._** (See [RELEASING.md](https://github.com/intersectmbo/cardano-ledger/blob/- [ ] When applicable, versions are updated in `.cabal` and `CHANGELOG.md` files according to the
```

Comment:
The markup is broken here. I think some text got deleted accidentally, because the link is incomplete and what is meant to be the following list item is at the end of the line.

Reply:
Good catch. Thank you